src/constraint_system/ecc/mod.rs

```diff
@@ -14,6 +14,7 @@ use ark_ec::{
     twisted_edwards_extended::GroupAffine, PairingEngine, TEModelParameters,
 };
 use core::marker::PhantomData;
+use std::ops::Neg;
```

Comment:
This can be removed and then the uses of `.neg()` be replaced by `-`.

Reply:
fixed